src/mempool_options.h

```diff
@@ -4,8 +4,13 @@
 #ifndef BITCOIN_MEMPOOL_OPTIONS_H
 #define BITCOIN_MEMPOOL_OPTIONS_H
 
+#include <cstddef>
+
 class CBlockPolicyEstimator;
 
+/** Default for -maxmempool, maximum megabytes of mempool memory usage */
+static const unsigned int DEFAULT_MAX_MEMPOOL_SIZE_MB = 300;
```

Comment:
nit: maybe make it (along with the expiry one) `constexpr`?

Reply:
Fixed in 52349ea72f58b4057a88890fe0c7b722698e3fac